### ISSUE TITLE
Remember layout 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#427]
```
What steps will reproduce the problem?
1. There is a horizontal pane above the notes contents called the "List View".  
Resize that pane to a zero height (effectively removing it)[1]
2. Close and reopen keepnote

What is the expected output? What do you see instead?
I'd expect that pane to remain at a zero height, but it comes back to a default 
height of a few lines.

What version of the product are you using? On what operating system?
KeepNote 0.7.1 on Linux and OS X

[1] Honestly, if there were an option to get rid of that pane altogether I'd 
use it instead.  I don't find the pane useful.  I can file another bug for that 
if it'll help.
```

Original issue reported on code.google.com by `clouserw` on 8 Mar 2011 at 8:54

**Comment by GoogleCodeExporter** [comment]
```
Thanks for submitting this bug.  Can you post a screenshot of what you see?  I 
cannot reproduce this behavior (for me zero size listview is preserved).

Also in the file ~/.config/keepnote/keepnote.xml what does the line with 
"hash_pos" say?  It should say something like:

<key>hsash_pos</key><integer>0</integer>

or some other integer.


```

Original comment by `matt.rasmus@gmail.com` on 8 Mar 2011 at 10:13

**Comment by GoogleCodeExporter** [comment]
```
Screenshot attached.  No matter what size I set that pane, my hsash_pas is 258, 
even after restarts:

~$ grep "hsash_pos" .config/keepnote/keepnote.xml
    <key>hsash_pos</key><integer>258</integer>
~$

```

Original comment by `clouserw` on 8 Mar 2011 at 10:29

Attachments:
- [layer_menu.png](https://storage.googleapis.com/google-code-attachments/keepnote/issue-427/comment-2/layer_menu.png)

**Comment by GoogleCodeExporter** [comment]
```
This will be fixed in keepnote 0.7.4.
```

Original comment by `matt.rasmus@gmail.com` on 19 Jun 2011 at 9:41
- Changed state: **Started**
- Added labels: **ui**

**Comment by GoogleCodeExporter** [comment]
```
I can verify this bug is fixed in 0.7.4 but can't close it here
```

Original comment by `clouserw` on 25 Jul 2011 at 4:03

**Comment by GoogleCodeExporter** [comment]
```
Thanks!
```

Original comment by `matt.rasmus@gmail.com` on 25 Jul 2011 at 4:22
- Changed state: **Done**
